code/go/0chain.net/smartcontract/zcnsc/authorizer.go

```diff
@@ -116,7 +117,22 @@ func (zcn *ZCNSmartContract) AddAuthorizer(
 	// Events emission
 	ctx.EmitEvent(event.TypeStats, event.TagAddAuthorizer, authorizerID, authorizer.ToEvent())
 
-	return string(authorizer.Encode()), nil
+	err = updateAuthorizerCount(ctx)
+
+	return string(authorizer.Encode()), err
+}
+
+func updateAuthorizerCount(ctx cstate.StateContextI) (err error) {
+	var numAuth item
+	err = ctx.GetTrieNode(storagesc.ALL_AUTHORIZERS_KEY, &numAuth)
+	if err != nil {
```

Comment:
Check if err is `util.ErrValueNotPresent` and return if it's not as we must return if the err is `ErrNodeNotFound`.